PostPairsTrading.md

```diff
@@ -227,33 +220,19 @@ This precisely meets one of our objectives: getting **a comprehensive method for
 
 ## Real-time spread calculation
 
-Now that we have selected a pair of cointegrated indices and understand how to calculate their relationships, let's see how we can create a real-time pair trading scenario.
-
-To do this, we need to focus on the Real-Time Pair Trading Subscriber (RPT). As can be seen in the diagram, this component receives 3 inputs.
-
-Let's start with the easy part: first, we receive which pair of indexes we have chosen in the previous step (this step, in our case, is done AD-HOC since, as we mentioned, we have chosen these two indices not only because of their cointegration but also because they are indices whose tick data is public).
-
-The next step is to receive the data for this pair of indices. For this, we need to subscribe to the tickerplant using the tick architecture function `.u.sub`. Here, we need to specify which table and which symbols (indexes in our case) we want to subscribe to. This information needs to be passed through the connection to the tickerplant. 
-
-> 💡 `.z.x` in the KDB+/Q language is an environment variable that stores the command-line arguments passed to a q script.
+Now that we have selected a pair of cointegrated indices and built a model to calculate their relationships, it's time to formalize its subscription as a real-time component. Once we start receiving data from the TP, we can apply the model to produce the spreads, which will then be sent to the dashboard.
+![Arch-bottom](resources/general-architecture-bottom.png)

```

Comment:
```suggestion

![Arch-bottom](resources/general-architecture-bottom.png)

```

Reply:
I think it's ok to integrate graphs in the very same paragraph, since they are not independent figures that we are referring to, but if you prefer to keep them in a single one, it's fine.